runbotics/runbotics-orchestrator-ui/src/src-app/translations/en/process/actions/folder.ts

```diff
@@ -1,15 +1,20 @@
 const folderActionsTranslations = {
-    'Process.Details.Modeler.Actions.Folder.Delete.Label': 'Delete folder',
+    'Process.Details.Modeler.Actions.Folder.Delete.Label': 'Delete',
     'Process.Details.Modeler.Actions.Folder.Delete.Path': 'Path',
     'Process.Details.Modeler.Actions.Folder.Delete.Name': 'Name',
     'Process.Details.Modeler.Actions.Folder.Delete.Recursive': 'Recursive',
     'Process.Details.Modeler.Actions.Folder.DisplayFiles.Label': 'Display files',
     'Process.Details.Modeler.Actions.Folder.DisplayFiles.Path': 'Path',
     'Process.Details.Modeler.Actions.Folder.DisplayFiles.Name': 'Name',
-    'Process.Details.Modeler.Actions.Folder.Create.Label': 'Create folder',
+    'Process.Details.Modeler.Actions.Folder.Create.Label': 'Create',
     'Process.Details.Modeler.Actions.Folder.Create.Name': 'Name',
     'Process.Details.Modeler.Actions.Folder.Create.Name.Info': 'New folder name',
     'Process.Details.Modeler.Actions.Folder.Create.Output.Info': 'Save path of the newly created folder to a variable',
+    'Process.Details.Modeler.Actions.Folder.Rename.Label': 'Rename',
+    'Process.Details.Modeler.Actions.Folder.Rename.Path.Info': 'Path, e.g. C:\\Documents\\OldFolderName',
+    'Process.Details.Modeler.Actions.Folder.Rename.NewName': 'New name',
+    'Process.Details.Modeler.Actions.Folder.Rename.NewName.Info': 'New name, e.g. NewFolderName',
```

Comment:
imo it suggest a little that the folder name should not contain whitespaces

Reply:
I added to the tooltip information about forbidden characters.